tests/test_dense.py

```diff
@@ -50,3 +58,120 @@ def test_adjoint(self, shape):
     def test_trace(self, matrix, trace):
         cupy_array = CuPyDense(matrix)
         assert cupy_array.trace() == trace
+
+
+def test_true_div(shape):
+
+    array = np.random.uniform(size=shape) + 1.0j * np.random.uniform(size=shape)
+
+    cup_arr = CuPyDense(array)
+    cpdense_tr = cup_arr / 2.0
+    qtpdense_tr = data.Dense(array) / 2.0
+
+    np.testing.assert_array_equal(cpdense_tr.to_array(), qtpdense_tr.to_array())
```

Comment:
```suggestion
    cpdense_over_2 = cup_arr / 2.0
    qtpdense_over_2 = data.Dense(array) / 2.0

    np.testing.assert_array_equal(cpdense_over_2.to_array(), qtpdense_over_2.to_array())
```
Fix copy and pasted names.